baselines/ippo/config.py

```diff
@@ -27,6 +27,12 @@ class ExperimentConfig:
     goal_achieved_weight: float = 1.0
     off_road_weight: float = 0.0
 
+    # RESAMPLE TRAFFIC SCENARIOS
+    resample_scenarios: bool = True
+    resample_criterion: str = "global_step"  # Options: "global_step"
+    resample_freq: int = 100_000  # Resample every k steps (recommended to be a multiple of num_worlds * n_steps)
```

Comment:
Think this is way too low of a default if resampling is as slow as y'all are suggesting it is

Reply:
True! Set it to 2 M as default